IM_calculation/Advanced_IM/overview_adv_IM_status.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import argparse
+import glob
+import pandas as pd
+from pathlib import Path
+import sys
+
+ALL_STATUS = [
+    "finished",
+    "not_started",
+    "not_converged",
+    "not_finished",
+    "timed_out",
+    "crashed",
+    "unknown",
+]
+TYPES = ["obs", "sim"]
+
+SIM_DIR_SUFFIX = "Runs/*/*/IM_calc/{}_status.csv"
+OBS_DIR_SUFFIX = "ObservedGroundMotions/IM_calc/*/{}_status.csv"
```

Comment:
Because of the assumed knowledge about sim_structure this script might be better off in sim_workflow.
it's generally assumed that im_calc, empirical_engine, visualization etc. don't know the cs folder structure, so all the folder structure code is in one place. Especially around the retrieval of the fault/rel_name from the folder structure.

Perhaps a better solution would be to pass all the *_status.csv filenames as a nargs=+ to this script and then add it to the slurm script for advanced IMs?

Thoughts @jpa198 ?

Reply:
I'm definitely against having to pass in the glob strings manually every time, when they're basically the same every time. This isn't an event/realisation level script, it's a sim root level script, so wouldn't fit into any automated workflow slurm script. (It could, but it would result in realisations interacting with each other which has been a no-go so far)
Having a third 'TYPES' entry for custom could be good if we have a scenario where the output folder structure doesn't match the current sim or obs structures.
Sending this script over to get lost in the scripts folder their seems like overkill too.

I think this script is fine where it is, if you add a custom TYPE and consider that the default, and the sim/obs to be commonly used shortcuts.

Reply:
I'm with @jpa198 and don't think this belongs to sim_workflow. This is specific to advanced IM.  A custom TYPE is an interesting idea, but it will take a bit of work to accommodate a random directory structure. Will get it implemented shortly

Reply:
I'd say that a custom type is a good idea, but it's probably not worth the development time for it currently. If there's a need we can implement it later.

If at all possible, can we abstract the sim_structure assumptions to qcore.sim_structure?

Reply:
For a script that doesn't even import qcore atm, that seems a bit over kill

Reply:
I agree - and the pattern is not too difficult to figure out when needed

Reply:
Would that even work? It'd have to be pretty jank

Reply:
Ok, if you give the realisation as '*' it would spit out the same, so it would work. Yeah, it probably is safest to import sim_struct instead of leaving the hardcoded in there, in case we change the locations in future or something

Reply:
As in path.glob(pattern), something that has "*" should be pattern, not path, I had to make a bit of rearrangement. This should make you both happy.

Reply:
custom type was also implemented